sdk/Makefile

```diff
@@ -0,0 +1,18 @@
+# Copyright 2025 The Machine Controller Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.PHONY: lint
+lint:
+	golangci-lint run --verbose --print-resources-usage ./...
+	golangci-lint run --verbose --print-resources-usage --config ../.golangci.yml ./...
```

Comment:
Why do we run golangci-lint two times? Can you leave a comment in the Makefile why do we do this if this is intentional?